### PR TITLE
Base Stats In Modal

### LINKED ISSUE #56: Base Stats in Modal
**Is your feature request related to a problem? Please describe.**
Need to display base stats and graphic in modal.

**Describe the solution you'd like**
Base stats with graph that represents pokemons stat base stat levels

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Implemented Chart.js
2. Display base stats for pokemon through a bar chart
3. Bar chart color changed according to pokemon type

## Purpose
To add feature so users can view pokemon base stats through the base stats tab in the modal

https://www.chartjs.org/

Closes #56

### CODE REVIEW

File: src/components/BaseStats.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+              title: {
+                display: false,
+                text: "",
+              },
+              tooltip: {
+                enabled: false,
+              },
+            },
+          }}
+        />
```
Comment (kmausolf) [inline src/components/BaseStats.js:82]: This is RAD.  Nice job finding this library!

File: src/components/Modal.js [hunk 2]
```diff
@@ -70,6 +72,8 @@ export const Modal = ({
             height={height}
             weight={weight}
             abilities={abilities}
+            stats={stats}
+            types={types}
```
Comment (BlackCubes) [inline src/components/Modal.js:76]: This modal is going to look interesting

**Review by BlackCubes — APPROVED** [review]
Everything looks good! Noice!

File: src/components/BaseStats.js (hunk 1 above)
Comment (JorgeLVilla) [inline src/components/BaseStats.js:82]: Definitely gonna have to try this library out at some point. Great job with this!

**Review by JorgeLVilla — APPROVED** [review]
Excited to see this in action! Great job!

File: src/components/BaseStats.js (hunk 1 above)
Comment (krferrell) [reply src/components/BaseStats.js:82]: Just looked through Chart.Js and it looks awesome! Definitely going to have to remember this one!

**Review by krferrell — APPROVED** [review]
Excited to see Chart.Js in action. The code looks nice and clean too.

**Review by Lap343 — APPROVED** [review]
Never seen this library but am definitely checking it out. Good work

File: src/components/BaseStats.js (hunk 1 above)
Comment (JayDampitan) [reply src/components/BaseStats.js:82]: This library looks really good. Good finds!

**Pull request merged**